templates/Devel.dockerfile.j2

```diff
@@ -86,6 +115,12 @@ numba.cuda cannot load that, and instead have it load
 ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH_PREBUILD}
 {% endif %}
 
+{# Report ccache stats and cleanup; also fix  #}
+RUN ccache -s
+RUN ccache -c \
+    && chmod -R ugo+w /ccache
+RUN ccache -s
```

Comment:
I would combine these into a single run and drop the `chmod` command here. If needed you can add echos, for context info.

Also I would move `/ccache` to the `chmod` command in `partials/cleanup-chmod.dockerfile.j2`. This way it is done all at once

Reply:
I combined the above into a single command and I hear the sentiment about having all the chmod cleanup together. The issue with moving the `chmod /ccache` is that the command will also be distributed to non-devel images which don't use ccache. Do you think this will be a problem?

Reply:
Fair point, I think there is a way to leverage the template-ing language to do an if statement, but what you have now if good enough